extensions/vscode/src/extension/VsCodeExtension.ts

```diff
@@ -360,15 +361,23 @@ export class VsCodeExtension {
         return uri.query;
       }
     })();
+
     context.subscriptions.push(
       vscode.workspace.registerTextDocumentContentProvider(
         VsCodeExtension.continueVirtualDocumentScheme,
         documentContentProvider,
       ),
     );
 
+    context.subscriptions.push(
+      vscode.window.onDidChangeTextEditorSelection((event) => {
+        const filepath = event.textEditor?.document?.uri.fsPath || null;
+        this.debouncedNotifyEditorChange(filepath);
+      })
+    );
```

Comment:
what is this event? is this when we select text in a file. if so, then maybe we don't want to trigger notifyeditorchange

Reply:
This even is specific to getting the file path of the selected file.